canonical-web-and-design/charmhub.io
# Add cookie policy module

## Done
Add cookie policy module

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Check you see the cookie policy module panel
- Open the manager and make a selection
- The panel should disappear and the `_cookies_accepted` cookie should be set
- Scroll to the footer and hit the "Manage your tracker settings" link
- This should bring up the cookei policy

## Issue / Card
Fixes https://github.com/canonical-web-and-design/charmhub.io/issues/472
